cmd/main.go

```diff
@@ -76,6 +79,8 @@ func Execute() {
 	cobra.OnInitialize(initLog)
 	rootCmd.PersistentFlags().StringSlice(tagsFlagName, []string{"all"}, "select rules to be applied")
 	rootCmd.PersistentFlags().String(logLevelFlagName, "info", "log level (trace, debug, info, warn, error, fatal)")
+	rootCmd.PersistentFlags().StringSlice(reportPath, []string{""}, "path to generate report file. Available formats are: json, yaml and sarif")
```

Comment:
none of them should be mandatory, isn't that what we agreed? Report File is only if the user wants a report and the stdoutFormat has the default value set to yaml so it will print yaml by default like we talked about

Reply:
What you say is it will always print to the `stdout` (because you can't omit it, if so, it will be `yaml` as default)?

OK then, let's keep it like that.

But please consider my first suggestion, I think we need to explain that the format will be decided by the file extension,